docs/how-to-guides/block-tutorial/writing-your-first-block-type.md

```diff
@@ -42,10 +42,10 @@ Create a basic `block.json` file there:
 ```json
 {
 	"apiVersion": 2,
-	"name": "gutenberg-examples/example-01-basic-esnext",
 	"title": "Example: Basic (ESNext)",
-	"icon": "universal-access-alt",
+	"name": "gutenberg-examples/example-01-basic-esnext",
```

Comment:
Thank you for the PR. I agree with you that it will make easier for the users to understand. Though not sure if this change is required or not. Let's wait for the others to comment.

Reply:
Thanks for the response @amustaque97. I'm not sure what you mean by "required", though. It's a (small) improvement, and there are no risks whatsoever, so I don't see any reason to not publish it.

Reply:
> Thanks for the response @amustaque97. I'm not sure what you mean by "required", though. It's a (small) improvement, and there are no risks whatsoever, so I don't see any reason to not publish it.

I agree there is no risk but there could be a chance someone wants to improve a few other things. or other possibility is that there might be some high priority  PR needs to be reviewed/merged.